engines/bastion_katello/app/assets/javascripts/bastion_katello/products/views/products.html

```diff
@@ -41,6 +41,10 @@ <h2 translate>Products</h2>
         </li>
 
         <li role="menuitem">
+          <a ng-click="openAdvancedSyncModal()" ng-show="permitted('edit_products')" translate>Advanced Sync</a>
```

Comment:
I know it adds an extra click; however, would we want to just have 'Sync Selected' go to the modal and present the user with the sync options?

We could have Optimized selected as the default, but would be a way for users to see all of the options before proceeding.

I am honestly Ok with the current updates, but mentioning as a possible alternative, since now might be the time to change it since non-nutupane hasn't been out for too long.